src/tools/miri/src/shims/time.rs

```diff
@@ -111,11 +111,12 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
     fn GetSystemTimeAsFileTime(
         &mut self,
         LPFILETIME_op: &OpTy<'tcx, Provenance>,
+        link_name: &str,
```

Comment:
That's a confusing argument name, maybe `shim_name` or so? I first through "link" refers to "symlink" or so.^^ Also please make this the first argument.